Audio test was built to access real audio hardware.
Patch disables audio test to pass all the other tests on CI and other no-audio-hardware systems.
---
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 52b0de2..bfb6d9c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -196,12 +196,6 @@ if((ENABLE_GAME OR ENABLE_TESTS) AND NOT MSVC)
 	target_link_libraries(sdl2_webp ${game-external-libs})
 	add_test(NAME SDL2_WEBP_SUPPORT COMMAND sdl2_webp "${CMAKE_SOURCE_DIR}/data/core/images/scons_conftest_images/end-n.webp")
 
-	# test for audio support in SDL2
-	add_executable(sdl2_audio conftests/sdl2_audio.cpp)
-	set_target_properties(sdl2_audio PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/Testing)
-	target_link_libraries(sdl2_audio ${game-external-libs})
-	add_test(NAME SDL2_AUDIO_SUPPORT COMMAND sdl2_audio "${CMAKE_SOURCE_DIR}/data/core/music/main_menu.ogg")
-
 	add_custom_target(conftests
 		COMMAND ${CMAKE_COMMAND} -E env "SDL_AUDIODRIVER=dummy" ${CMAKE_CTEST_COMMAND}
 		DEPENDS
@@ -211,7 +205,6 @@ if((ENABLE_GAME OR ENABLE_TESTS) AND NOT MSVC)
 			sdl2_jpg
 			sdl2_png
 			sdl2_webp
-			sdl2_audio
 	)
 endif()
 
